inc/thorvg.h

```diff
@@ -210,6 +210,8 @@ class TVG_EXPORT Shape final : public Paint
 public:
     ~Shape();
 
+    Result duplicate(std::unique_ptr<tvg::Shape>& shape);
```

Comment:
Why not this?
std::unique_ptr<tvg::Shape> duplicate() const noexcept;

Reply:
I was thought that it will be easier to integrate with EFL. Duplication there looks like example below, so 
If we call duplicate which creates new object, we have to replace pointers and delete old object, but of course 
I can do it. Please let me know if you want it 
```c
Efl_VG *new_shape = evas_vg_shape_add(...); 
evas_vg_shape_dup(new_shape, old_shape)
```

Reply:
I stick to use  std::unique_ptrtvg::Shape duplicate() const noexcept;   this style, 
it's better for current thorvg API concept wise. Can you please change it?

Reply:
Done